package.json

```diff
@@ -8,7 +8,7 @@
     "flow": "flow; test $? -eq 0 -o $? -eq 2",
     "build": "yarn upgrade && browserify client/sync.js | uglifyjs - > bundles/bundle.js",
     "browsertest": "yarn start-test; browserify test/client/*.js test/*.js | uglifyjs - | tape-run; yarn stop-test",
-    "browsertest-client": "yarn start-test; browserify test/client/*.js | uglifyjs - | tape-run --browser chrome; yarn stop-test",
```

Comment:
yes; browsertest-client is helpful for developing client-only code because it runs faster. it's only used in dev, so it's ok to skip uglify as long as you check browsertest before committing.